src/views/v2/components/events-bar/search/keyword.php

```diff
@@ -38,4 +38,11 @@ class="tribe-common-form-control-text__input tribe-events-c-search__input tribe-
 		placeholder="<?php echo esc_attr( $placeholder ); ?>"
 		aria-label="<?php echo esc_attr( $aria_label ); ?>"
 	/>
+	<em
+		class="tribe-events-c-search__input-control-icon"
+		aria-label="<?php esc_attr_e( 'Search', 'the-events-calendar' ); ?>"
+		title="<?php esc_attr_e( 'Search', 'the-events-calendar' ); ?>"
+	>
+		<?php $this->template('components/icons/search', [ 'classes' => ['tribe-events-c-search__input-control-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: bb24812e2e73e4649a1b08eb689875a1</sub>